application/frontend/src/pages/releases/detail/cases-box/consent-summary.tsx

```diff
@@ -84,29 +85,22 @@ function ConsentSummary({ consentId, releaseKey, nodeId }: Props) {
 
   const terminologyFhirUrl = envRelay.terminologyFhirUrl;
 
-  // Url to fetch depending on what is given from props.
-  let u = "";
+  let consentQuery;
   if (consentId) {
-    u = `/api/datasets/consent/${consentId}`;
+    consentQuery = trpc.datasetRouter.getDatasetConsent.useQuery({ consentId });
   } else {
-    u = `/api/releases/${releaseKey}/consent/${nodeId}`;
+    consentQuery = trpc.releaseRouter.getReleaseConsent.useQuery({
+      releaseKey: releaseKey ?? "",
+      nodeId: nodeId ?? "",
+    });
   }
 
+  const duosCode = (consentQuery.data ?? []) as DuoLimitationCodedType[];
```

Comment:
This is feedback only on style, so feel free to ignore: Type annotations are preferable to type assertions:

```suggestion
  const duosCode: DuoLimitationCodedType[] = consentQuery.data ?? [];
```

Annotations give TypeScript the opportunity to check the types' correctness but assertions don't. It doesn't make any difference in this case, because TypeScript doesn't know what `consentQuery.data` is (it thinks it's `any`), but if the code were ever refactored such that `consentQuery.data`'s type was something else, it's be nice to give TypeScript the opportunity to call-out potential problems